code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -119,6 +119,13 @@
 	if(damagetype_healer == "all")
 		return
 
+	AddComponent(/datum/component/tippable, \
+		tip_time = 3 SECONDS, \
+		untip_time = 3 SECONDS, \
+		self_right_time = 3.5 MINUTES, \
```

Comment:
The `self_right_time` here is currently eyeballed since the panic proc is done on a processing subsystem, not sure if there's a cleaner way of going about this

Reply:
Is your concern consistency with the old behavior?

Reply:
Prior the medibot rights itself after it reaches 100 panic (`MEDBOT_PANIC_END`), which was roughly 3.5 minutes as far as I could tell - but since panic was handled by the processing from NPC pool it's probably not a one to one timer